### PR TITLE
Moved botkit shell to dev dependency

### PR DESCRIPTION
**Pull request by silentworks**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = {
   RANDOM_ID: process.env.randomId || '',
   EVENTS_ID: process.env.eventsId || '',
   ICON_URL: process.env.iconUrl || '',
+  API_AI_TOKEN: process.env.API_AI_TOKEN || 'pass your own key at runtime',
```
Comment (tonymobster) [inline config.js:12]: I discussed using https://www.npmjs.com/package/dotenv for this with @oddoye-david

File: interactive-shellbot.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    token=<MY TOKEN> API_AI_TOKEN=<MY AI TOKEN> node interactive-shellbot.js
+
+# EXTEND THE BOT:
+  Botkit has many features for building cool and useful bots!
+  Read all about it here:
+    -> http://howdy.ai/botkit
+*/
+'use strict';
+
+// Interactive shell for bot
```
Comment (tonymobster) [inline interactive-shellbot.js:16]: Lets set an environment variable and switch to shellbot for "Development" and the live one for "Production" instead.

Comment (silentworks) [reply interactive-shellbot.js:16]: @tonymobster you want them to live in the same index.js file?

Comment (tonymobster) [reply interactive-shellbot.js:16]: Yeah so the command to start it becomes the same for both production and development. Or I guess we could also just add another script to the `package.json` to enable running with `npm run shell`

**Pull request merged**